package.json

```diff
@@ -17,8 +23,8 @@
     "/build"
   ],
   "repository": {
-    "type" : "git",
-    "url" : "https://github.com/medhatdawoud/react-change-highlight"
+    "type": "git",
+    "url": "https://github.com/medhatdawoud/react-change-highlight"
```

Comment:
Try to use `prettier` to have the same formatting, just install plugin into your IDE or Editor, and use the ready config one

Reply:
I already use  prettier and didn't notice this change 👌👌